Demo/MapFilter/DemoSearchLocationDataSource.swift

```diff
@@ -47,28 +35,17 @@ class DemoSearchLocationDataSource: NSObject, SearchLocationDataSource {
     ]
 
     func searchLocationViewController(_ searchLocationViewController: SearchLocationViewController, didRequestLocationsFor searchQuery: String, completion: @escaping ((SearchLocationDataSourceResult) -> Void)) {
-        if #available(iOS 9.3, *) {
```

Comment:
I think it should be the other way around. We should keep this if statement because `#available(iOS 9.3, *)` means it is available on iOS10.

Reply:
of course, you are right. :)